opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -60,3 +61,33 @@ def unwrap(obj, attr: str):
     func = getattr(obj, attr, None)
     if func and isinstance(func, ObjectProxy) and hasattr(func, "__wrapped__"):
         setattr(obj, attr, func.__wrapped__)
+
```

Comment:
Would this logic be better off [here](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/util/opentelemetry-util-http)?

Reply:
Only thought I had was that it is most similar to the other utility function `http_status_to_status_code` in the same file and the library you linked is only used by web server instrumentations. However, it might make more logical sense to separate it from the `opentelemetry-instrumentation` itself. What would you suggest? @lzchen

Reply:
I believe the library would be used for all http related instrumentations, it just so happens the logic contained is only used in server instrumentations.

Reply:
Sounds good 👍 Just updated with the last remaining suggestions.